.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "markdown.extension.toc.levels": "2..6",
```

Comment:
Should thisfile be added to .gitignore?

Reply:
Not everyone uses VSCode but if you do, without the markdown extension, it'll mess up everything without this configuration 😢 on the bright side, it's actually a pretty handy extension and TOC generation is quite nice, so I'd like to keep this here if thats ok 😋

Reply:
Er, what do you mean "mess up"? I'm just more concerned with messing with VSCode settings for people 😛

Reply:
> Er, what do you mean "mess up"?

by default, it generates TOC with `-` and includes top-level `#` headers that we don't want (for us, they're page titles), and does it on save automatically

>  I'm just more concerned with messing with VSCode settings for people

this is only a workspace setting, so it won't do anything outside of `docs`

Reply:
Okay fair enough.